raiden-ts/src/logger.ts

```diff
@@ -7,9 +7,9 @@ Object.defineProperty(logging, 'methodFactory', {
   get() {
     return curMethodFactory;
   },
-  set(func: typeof curMethodFactory) {
+  set(func: typeof curMethodFactory & { allow_overwrite?: boolean }) {
     /* prevent anyone else but us (e.g. matrix) from overwriting methodFactory  */
-    if (!func.name?.startsWith('raiden')) return;
+    if (!func.allow_overwrite) return;
```

Comment:
Please, also add this property to the `cli`'s logger, or it won't work there.

Reply:
Ah damn. I checked the SDK for any modification, but forgot the CLI. Thanks for reminding me!

Reply:
✔️